src/linux/mod.rs

```diff
@@ -157,19 +144,18 @@ impl<'a> SegmentTrait for Segment<'a> {
     fn name(&self) -> &str {
         unsafe {
             match self.phdr.as_ref().unwrap().p_type {
-                PT_NULL => "NULL",
-                PT_LOAD => "LOAD",
-                PT_DYNAMIC => "DYNAMIC",
-                PT_INTERP => "INTERP",
-                PT_NOTE => "NOTE",
-                PT_SHLIB => "SHLI",
-                PT_PHDR => "PHDR",
-                PT_TLS => "TLS",
-                PT_NUM => "NUM",
```

Comment:
`PT_NUM ` means the number of defined program header types, it's not a real type.